modules/govuk/manifests/node/s_calculators_frontend.pp

```diff
@@ -29,6 +29,10 @@
     listen_ip  => '0.0.0.0',
   }
 
+  govuk_envar {
```

Comment:
Should this be `govuk_envvar` instead of `govuk_envar`?

Reply:
I keep doing this